packages/remark-mdx/test/fixtures/character-reference.out

```diff
@@ -4,15 +4,13 @@ The characters `>` and `}` don’t need to be handled differently: >, }.
 
 Those characters are also fine in JSX elements: <>> and }</>.
 
-{
-  /* starts an expression, it must be closed: */
-}
+{/* starts an expression, it must be closed: */}
 
-If you don’t want an expression, encode it, like so: &#x7B;.
+If you don’t want an expression, encode it, like so: \{.
```

Comment:
Escapes were a bit broken in old markdown and you had to resort to character references (“entities”). Since CommonMark, backslash escapes can now be used for all punctuation.